go-controller/pkg/cni/cni.go

```diff
@@ -93,24 +103,43 @@ func (pr *PodRequest) cmdAdd() *PodResult {
 		return nil
 	}
 
-	ovnAnnotation, ok := annotation["ovn"]
-	if !ok {
-		logrus.Errorf("failed to get ovn annotation from pod")
-		return nil
-	}
+	var ovnAnnotation string
+	var ok bool
+	var ovnNetworkAnnotatedMap map[string]string
+	if isDefaultInterface {
+		ovnAnnotation, ok = annotation[ovnAnnotationKey]
+		if !ok {
+			logrus.Errorf("failed to get ovn annotation from pod")
+			return nil
+		}
 
-	var ovnAnnotatedMap map[string]string
-	err = json.Unmarshal([]byte(ovnAnnotation), &ovnAnnotatedMap)
-	if err != nil {
-		logrus.Errorf("unmarshal ovn annotation failed")
-		return nil
+		err = json.Unmarshal([]byte(ovnAnnotation), &ovnNetworkAnnotatedMap)
+		if err != nil {
+			logrus.Errorf("unmarshal ovn annotation failed")
```

Comment:
You should also log the error message:

logrus.Errorf("unmarshal ovn annotation failed: %v", err)

Reply:
done